global_planner: "global_planner/GlobalPlanner"
allow_unknown: false
default_tolerance: 0.0
visualize_potential: false
use_dijkstra: false
use_quadratic: false
use_grid_path: true
old_navfn_behaviour: false
